cmd/dm/dm.go

```diff
@@ -272,7 +272,7 @@ func callService(path, method, action string, reader io.ReadCloser) {
 	var URL *url.URL
 	URL, err := url.Parse(*service)
 	if err != nil {
-		panic(fmt.Errorf("cannot parse url (%s): %s\n", *service, err))
+		panic(fmt.Errorf("cannot parse url (%s): %s\n", path, err))
```

Comment:
This diff is from reverting commit https://github.com/kubernetes/deployment-manager/commit/6e1821c4bb1c0b927fb99fa5ac1325bf41267db7